### ISSUE TITLE
Copters

### ISSUE THREAD

**Issue by mikeal** [mikeal/nodeconf2014#21]
For a variety of reasons I don't think a Copter "session" will work again this year.

Rather, I have a better idea.

On the first day of content let's do something formal for the Kids Track with the Copters while the attendees are busy doing all the sessions.

On the second day where people are choosing what they want to be engaging with we keep access and help alive all day for anyone who wants to play with the copters.

@aulvi and @nexxy what do you think?

@lyle what do you think about the Kids Track?

**Comment by aulvi** [comment]
Sounds great, I'm down to help with the "Copter Kids" track!

I think it would be fun to have groups try to fly an obstacle course in a Logo-ish manner (like, "fly 10 paces, turn right 90 degrees, fly 5 paces, turn left 90 degrees" etc).

Second day sounds pretty chill, we could easily keep some usb sticks hanging around with simple sample apps (gathered from wherever) and allow people to begin hacking quickly.

Anyways, I'm happy to help any way I can, these events are always a blast. Thanks!

**Comment by affinity-matrix** [comment]
just saw this thread :)

I also think this sounds good. If you are anticipating an increased youth presence this year, I think it would be particularly well suited. Last year there were at least a few kids that seemed _very_ interested in the copters, but I think apprehension took hold of either them or their parents.

If it was clearly communicated that copters and the like are specifically designated as kid-friendly (provided the parents are OK with the inherent risks involved), I would expect to see more involvement!

**Comment by mikeal** [comment]
I think we're just going to have part of the SO and Kids track on Friday come in to the barn for copter time. The regular attendees will be in sessions during that time so they'll have free reign over all flying objects :)

**Comment by lyle** [comment]
Yes, yes. Kids copter time on Friday. I'll help with that and bring kids.
We can get them in groups to do the coding and maybe write a routine to
show the parents at the end of the day.

**Comment by aulvi** [comment]
I love the idea of building a small obstacle course and having kids program
their drones to fly it. Maybe I'm just projecting my own desires, hard to
say ;)
